### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by izeye**
This PR fixes some typos and polishes trivial stuff.

### CODE REVIEW

File: spring-kafka/src/main/java/org/springframework/kafka/core/DefaultKafkaConsumerFactory.java [hunk 1]
```diff
@@ -121,18 +121,16 @@ public void setValueDeserializer(Deserializer<V> valueDeserializer) {
...
+		Map<String, Object> modifiedConfigs = new HashMap<>(this.configs);
+		if (groupId != null) {
+			modifiedConfigs.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
 		}
+		if (shouldModifyClientId) {
+			modifiedConfigs.put(ConsumerConfig.CLIENT_ID_CONFIG,
+				(overrideClientIdPrefix ? clientIdPrefix
+						: modifiedConfigs.get(ConsumerConfig.CLIENT_ID_CONFIG)) + clientIdSuffix);
+		}
+		return createKafkaConsumer(modifiedConfigs);
```
Comment (garyrussell) [inline spring-kafka/src/main/java/org/springframework/kafka/core/DefaultKafkaConsumerFactory.java]: What is the purpose of this change - seems like just a style preference.

File: spring-kafka/src/main/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainer.java [hunk 2]
```diff
@@ -99,14 +101,9 @@ public void setConcurrency(int concurrency) {
...
-		this.containers.forEach(c -> {
-			Collection<TopicPartition> assignedPartitions = c.getAssignedPartitions();
-			if (assignedPartitions != null) {
-				assigned.addAll(assignedPartitions);
-			}
-		});
-		return assigned;
+		return this.containers.stream()
+				.map(KafkaMessageListenerContainer::getAssignedPartitions).filter(Objects::nonNull)
+				.flatMap(assignedPartitions -> assignedPartitions.stream()).collect(Collectors.toList());
```
Comment (garyrussell) [inline spring-kafka/src/main/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainer.java]: we prefer
    
    .oneOperation()
    .perLine()
    .style()

File: spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java [hunk 3]
```diff
@@ -192,16 +192,11 @@ public void setClientIdSuffix(String clientIdSuffix) {
...
 			}
-			else {
-				return null;
-			}
-		}
-		else {
-			return null;
 		}
+		return null;
 	}
```
Comment (garyrussell) [inline spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java]: Again; style only.

File: spring-kafka/src/main/java/org/springframework/kafka/core/DefaultKafkaConsumerFactory.java (hunk 1 above)
Comment (izeye) [reply spring-kafka/src/main/java/org/springframework/kafka/core/DefaultKafkaConsumerFactory.java]: @garyrussell I think `if` block with `return` statement makes the following `else` block redundant and removing it reduces a block depth. WDYT?

Comment (artembilan) [reply spring-kafka/src/main/java/org/springframework/kafka/core/DefaultKafkaConsumerFactory.java]: I can't find the rule now, but we really prefer `if...else` style even if there is a `return` operator.

**Comment by izeye** [comment]
@garyrussell @artembilan Thanks for the feedback!

The requested changes have been applied in b9b54f7.

**Comment by garyrussell** [comment]
Sounds good; thanks - and thanks to @izeye
